### PR TITLE
Addition of point source containment correction for spectrum

### PR DESCRIPTION
**Pull request by registerrier**
Note that the psftable.integral is returning errors at several energies because of interpolation issues. This is solved by introducing a try/except and putting nan in the corrected arf. This results is some missing values in the arf.

Also removed the column bin_lo and bin_hi in the PHA file because they introduce some mess in sherpa

### CODE REVIEW

**Comment by cdeil** [comment]
@joleroi - Ready to merge?

File: gammapy/spectrum/tests/test_spectrum_fit.py [hunk 1]
```diff
@@ -10,6 +10,10 @@
 from ...utils.testing import requires_dependency, requires_data, SHERPA_LT_4_8
 from astropy.utils.compat import NUMPY_LT_1_9
 
+
+#TODO : PHACount should not expect a BIN_LO and BIN_HI columns
+#https://travis-ci.org/gammapy/gammapy/jobs/136170992#L939
```
Comment (cdeil) [inline gammapy/spectrum/tests/test_spectrum_fit.py]: Tests on travis-ci passed. @registerrier wants to start new work that partly builds on this pull request.
So the easiest would be to merge this PR now, if that's OK with you @joleroi ?
The main concern I have is that this PR adds an `xfail` to `test_spectral_fit` which I know you've been working on.
Maybe this has to wait a bit and then the xfail has to be removed here before it should go in?

Comment (joleroi) [inline gammapy/spectrum/tests/test_spectrum_fit.py]: No thats's fine. OK to merge. I will fix the test that had to be marked as xfail.

**Comment by cdeil** [comment]
Merging this now ...
Thanks, @registerrier!

**Pull request merged**